tests/jointcalTestBase.py

```diff
@@ -117,9 +117,14 @@ def _testJointCalTask(self, nCatalogs, dist_rms_relative, dist_rms_absolute, pa1
         oldWcsList = result.resultList[0].result.oldWcsList
 
         rms_result = self.jointcalStatistics.compute_rms(data_refs, self.reference)
+
+        # Make plots before testing, if requested, so we still get plots if tests fail.
+        if self.do_plot:
+            plot_dir = os.path.join('.test', self.__class__.__name__, 'plots')
+            if not os.path.isdir(plot_dir):
+                os.mkdir(plot_dir)
+            self.jointcalStatistics.make_plots(data_refs, oldWcsList, name=caller, outdir=plot_dir)
```

Comment:
I haven't run this particular test; did it save the plots somewhere else before or just open them up interactively? If the latter, or if the saved location is notably different from the old one, it might be helpful to explicitly tell the user where the plots have been saved.